examples/06_command_framework/src/main.rs

```diff
@@ -113,6 +113,8 @@ fn main() {
                 .desc("Sends an emoji with a dog.")
                 .bucket("emoji")
                 .exec_str(":dog:")))
+        .alias("Ungrouped", "multiply", "*")
+        .alias("Emoji", "cat", "kitty")
```

Comment:
I'm really unsure about this syntax. Maybe we need to make the Framework into a builder, and then have Client::with_framework call `build()` on the builder to produce a finalized Framework. I think this would open up a lot of possibilities WRT configuration of everything and - for example - should make aliasing in CreateGroups possible directly, `|group| group.alias("multiply", "*")`.

This is a story for another day though, I guess.